packages/app/package.json

```diff
@@ -8,17 +8,20 @@
 		"@package/scanner": "workspace:*",
 		"@package/ui": "workspace:*",
 		"@package/utils": "workspace:*",
+		"@paradym/wallet-sdk": "workspace:*",
 		"@react-native-community/netinfo": "catalog:",
 		"@tamagui/animations-react-native": "catalog:",
 		"@tamagui/colors": "catalog:",
 		"@tamagui/font-inter": "catalog:",
 		"@tamagui/lucide-icons": "catalog:",
+		"@tamagui/portal": "catalog:",
 		"@tamagui/shorthands": "catalog:",
 		"@tamagui/themes": "catalog:",
 		"@tanstack/react-query": "catalog:",
 		"expo-constants": "catalog:",
 		"expo-haptics": "catalog:",
 		"expo-linking": "catalog:",
+		"expo-modules-autolinking": "catalog:",
```

Comment:
Do we need this? I think this is already installed by expo (i removed it in a recent PR)